src/devices/src/virtio/balloon/utils.rs

```diff
@@ -270,4 +270,58 @@ mod tests {
             RemoveRegionError::MmapFail(_)
         );
     }
+
+    /// -------------------------------------
+    /// BEGIN PROPERTY BASED TESTING
+    use proptest::prelude::*;
+
+    fn random_pfn_u32_max() -> impl Strategy<Value = Vec<u32>> {
+        // Create a randomly sized vec (max 16k) filled with random u32 elements.
```

Comment:
Update the comment with the new size (`MAX_PAGE_COMPACT_BUFFER`).

The next comment as well.

Reply:
done